example/.watchmanconfig

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "prefer_split_fsevents_watcher": true
```

Comment:
Gets rid of the annoying re-crawled message when metro starts

Reply:
That always annoyed me, lol. I wonder why that bugs tho

Reply:
Tzvetan mentioned something about macos not being their default platform, so fsevents is kinda just duck-taped in